schemas/com.snowplowanalytics.snowplow/cookie_extractor_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for cookie extractor context generated by cookie extractor enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "cookie_extractor_context",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"cookies": {
+			"type": "object",
+			"properties": {
+				"name": "string",
+				"value": "string"
+			},
+			"required": ["name", "value"]
+		}
+	},
+	"required": ["cookies"],
+	"additionalProperties": false
+}
```

Comment:
I think there's one level too many here. Rather than

```json
{
  "cookies": {
    "name": "x",
    "value": "y"
  }
}
```

Shouldn't it just be

```json
{
  "name": "x",
  "value": "y"
}
```

?